src/sourcekit-lsp/getReferenceDocument.ts

```diff
@@ -22,7 +22,7 @@ export function activateGetReferenceDocument(client: langclient.LanguageClient):
         {
             provideTextDocumentContent: async (uri, token) => {
                 const params: GetReferenceDocumentParams = {
-                    uri: uri.toString(true),
+                    uri: uri.toString(),
```

Comment:
Best to use `code2protocolConverter` to convert between vscode and lsp types. See below